protocol/scripts/deploys/v2/sepolia/99-post-deployment.ts

```diff
@@ -0,0 +1,123 @@
+import { ethers } from 'hardhat';
+import {
+    ensureExpectedEnvvars,
+    mine,
+} from '../../helpers';
+import { connectToContracts, getDeployedContracts } from '../sepolia/contract-addresses';
+import { keccak256 } from 'ethereumjs-util';
+import { ITempleStrategy } from '../../../../typechain';
+
+async function main() {
+    ensureExpectedEnvvars();
+  
+    const [owner] = await ethers.getSigners();
+    const TEMPLE_V2_DEPLOYED = getDeployedContracts();
+    const templeV2contracts = connectToContracts(owner);
+    const EXTERNAL_ALL_USERS  = keccak256(Buffer.from("EXTERNAL_USER"));
+
+    // Setup Circuit Breaker for all users
+    {
+        await mine(templeV2contracts.circuitBreakerProxy.setIdentifierForCaller(owner.getAddress(), "EXTERNAL_USER"));
```

Comment:
owner isn't right here.

This is to set the identifier for whoever is going to call the circuit breaker proxy. This won't be the owner address, it will be:

```
setIdentifierForCaller(TEMPLE_V2_DEPLOYED.TEMPLE_LINE_OF_CREDIT.ADDRESS, "EXTERNAL_USER");
```

When we get around to adding Ramos and other strategies (like Gnosis strategies), they'll also need to be setup too.

We might want to consider splitting the post-deployment into:
   1/ 99a-v2-core-post-deployment.ts
   2/ 99b-tlc-post-deployment.ts
   3/ ...etc

Reply:
please double check this on my latest commits, I think I may have done something wrong here